polaris-react/src/components/Popover/Popover.stories.tsx

```diff
@@ -25,26 +25,51 @@ export default {
 } as ComponentMeta<typeof Popover>;
 
 export function WithActionList() {
-  const [popoverActive, setPopoverActive] = useState(true);
+  // const [popoverActive, setPopoverActive] = useState(true);
+  const [activePopover, setActivePopover] = useState(null);
 
-  const togglePopoverActive = useCallback(
-    () => setPopoverActive((popoverActive) => !popoverActive),
-    [],
-  );
+  const togglePopoverActive = useCallback((popover, isClosing) => {
+    const ap = isClosing ? null : popover;
```

Comment:
Question: what is `ap`? Can we use a more explicit variable name?

Reply:
activePopover was already taken, so ap was shorthand. I changed it to currentPopover which seemed like the closest next option.